dotCMS/src/main/java/com/dotmarketing/fixtask/tasks/FixTask00095DeleteOrphanRelationships.java

```diff
@@ -0,0 +1,199 @@
+            "(SELECT * FROM structure WHERE relationship.parent_structure_inode = structure.inode) " +
+            " OR NOT EXISTS " +
+            "(SELECT * FROM structure WHERE relationship.child_structure_inode = structure.inode) ";
+
+    @Override
+    public List<Map<String, Object>> executeFix() throws DotDataException {
+        Logger.info(FixTask00095DeleteOrphanRelationships.class,
+                "Beginning DeleteOrphanRelationships");
+        List<Map<String, Object>> returnValue = new ArrayList<>();
+
+        if (!FixAssetsProcessStatus.getRunning()) {
+            HibernateUtil.startTransaction();
+            int total = 0;
+            try {
+                FixAssetsProcessStatus.startProgress();
+                FixAssetsProcessStatus
+                        .setDescription("task 95: DeleteOrphanRelationships");
+                DotConnect dc = new DotConnect();
+                dc.setSQL(VERIFICATION_QUERY);
+                modifiedData = dc.loadResults();
+                total = modifiedData != null ? modifiedData.size() : 0;
+                FixAssetsProcessStatus.setTotal(total);
+                getModifiedData();
+                if (total > 0) {
+                    startCleanup();
+                    saveFixAuditRecord(total);
+                    returnValue.add(FixAssetsProcessStatus.getFixAssetsMap());
+                    Logger.debug(
+                            FixTask00095DeleteOrphanRelationships.class,
+                            "Ending DeleteOrphanRelationships");
+                }
+            } catch (DotHibernateException e) {
+                Logger.debug(
+                        FixTask00095DeleteOrphanRelationships.class,
+                        "There was a problem during DeleteOrphanRelationships",
+                        e);
+                HibernateUtil.rollbackTransaction();
+                FixAssetsProcessStatus.setActual(-1);
+            } catch (Exception e) {
+                Logger.debug(
+                        FixTask00095DeleteOrphanRelationships.class,
+                        "There was a problem during DeleteOrphanRelationships",
+                        e);
+                HibernateUtil.rollbackTransaction();
+                FixAssetsProcessStatus.setActual(-1);
+            } finally {
+                FixAssetsProcessStatus.stopProgress();
+            }
+        }
+        return returnValue;
+    }
+
+    @Override
+    public List<Map<String, String>> getModifiedData() {
+        if (modifiedData.isEmpty()) {
+            XStream xstreamObj = new XStream(new DomDriver());
+            LocalDate date = LocalDate.now();
+            SimpleDateFormat sdf = new SimpleDateFormat("dd-MM-yyyy_HH-mm-ss");
+            String lastmoddate = sdf.format(date);
+            File writingObj = null;
+            if (!new File(ConfigUtils.getBackupPath() + File.separator
+                    + "fixes").exists()) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "fixes" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)